suite/tests/api/ir_x86.c

```diff
@@ -787,6 +787,46 @@ test_instr_encode_and_decode(void *dc, instr_t *instr, uint len_expect,
     instr_destroy(dc, decin);
 }
 
+bool
+instr_same_synthetic(instr_t *inst1, instr_t *inst2)
```

Comment:
I think we should have a separate api/ir_synthetic.c for these tests.

Reply:
Agreed.
Will do, but for now I just wanted to see if it works.

Reply:
Done.